web3/embed.html

```diff
@@ -19,12 +19,15 @@
       .loading {
         position: absolute;
         bottom: 0;
-        right: 0;
+        right: 1em;
         text-align: right;
-        font-size: 1.3em;
+        font-size: 1.2em;
         color: #fff;
-        background: #e66;
-        padding: .5em .5em .2em;
+        background: rgba(40, 40, 40, 0.75);
+        padding: 0 1em;
+        height: calc(1em + 36px);
```

Comment:
The same div is used to display information about unsupported version, does it look good with it too? (To test you need to build `beta` branch)